TypeCobol/Compiler/SqlNodes/Statements.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Nodes;
+using TypeCobol.Compiler.SqlCodeElements;
+using TypeCobol.Compiler.SqlCodeElements.Statement;
+
+namespace TypeCobol.Compiler.SqlNodes
+{
+    /// <summary>
+    /// Sql COMMIT Node
+    /// </summary>
+    public class Commit : GenericNode<CommitStatement>, TypeCobol.Compiler.Nodes.Statement
```

Comment:
Qualifier is redundant

Reply:
done in [8e7eb2d](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/8e7eb2d5d8099a08412b896ce27756a1a43c4800)